csharp/src/Google.Protobuf/CodedOutputStream.cs

```diff
@@ -160,6 +160,33 @@ public long Position
             }
         }
 
+        /// <summary>
+        /// Configures serialization to be deterministic.
```

Comment:
Perhaps "Configures whether or not serialization is deterministic." (It's a property, but the current text sounds sounds like it's a method.)